Demo-Android/gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-agp = "8.4.2"
+agp = "8.5.1"
```

Comment:
The AGP and Gradle versions were updated to mirror the WordPress-Android project.